backend/app/notify/email_test.go

```diff
@@ -74,9 +74,9 @@ func TestEmailNew(t *testing.T) {
 				assert.NoError(t, err)
 				assert.NotNil(t, email, "email returned")
 
+				assert.Equal(t, defaultEmailTemplatePath, email.EmailParams.MsgTemplatePath, "empty emailParams.MsgTemplatePath changed to default")
+				// assert.Equal(t, defaultEmailVerificationTemplatePath, email.EmailParams.VerificationTemplatePath, "empty emailParams.VerificationTemplatePath changed to default")
```

Comment:
Why this is commented out?

Reply:
I've checked, it should not: it's working check.

Reply:
Oh, I skipped it.